libp2p/peerinfo.nim

```diff
@@ -29,6 +29,7 @@ type
   PeerInfo* = ref object of RootObj
     peerId*: PeerID
     addrs*: seq[MultiAddress]
+    secureCodec*: string
```

Comment:
This will not hold, if the peer has several connections from different transports, they could have been secured over different channels. Same for inbound/outbound connections, they could have been secured over different mechanisms.

Reply:
right